anndata/_core/anndata.py

```diff
@@ -594,14 +594,20 @@ def _init_as_actual(
 
     def __sizeof__(self, show_stratified=None) -> int:
         def get_size(X):
-            if issparse(X):
-                X_csr = csr_matrix(X)
+            def csr_to_bytes(X_csr):
```

Comment:
The code should also handle CSC matrices.

```suggestion
            def cs_to_bytes(X: sparse.csr_matrix | sparse.csc_matrix):
```

Reply:
Noted. Changes Applied